src/routes/index.ts

```diff
@@ -118,6 +122,20 @@ router.get(config.MANAGING_OFFICER_CORPORATE_URL + config.REMOVE + config.ID, au
 router.get(config.TRUST_INFO_URL, authentication, navigation.hasBOsOrMOs, trustInformation.get);
 router.post(config.TRUST_INFO_URL, authentication, navigation.hasBOsOrMOs, ...validator.trustInformation, checkTrustValidations, trustInformation.post);
 
+if (isActiveFeature(FEATURE_FLAG_ENABLE_TRUSTS_WEB)) {
```

Comment:
Use middleware chained for this endpoint or add it on the controller, keep this file clean

Reply:
What is benefit of middleware?